backend/database/repositories/main.go

```diff
@@ -15,7 +15,7 @@ const (
 )
 
 // The ID for root, set this as the ID in a specified request
-const FILESYSTEM_ROOT_ID = 0
+const FILESYSTEM_ROOT_ID string = "00000000-0000-0000-0000-000000000000"
```

Comment:
I'll add a general comment later but I think this is a good example, IMO it would be better if we surfaced these primary keys are UUID types as opposed to strings, checkout this library for a nice uuid type + relevant operations: https://github.com/google/uuid

but by using this library we can turn this line into:
```go
const FILESYSTEM_ROOT_ID uuid.UUID = uuid.Nil 
```